lib/secure-boot-lib-dasharo.robot

```diff
@@ -107,6 +107,7 @@ Make Sure That Keys Are Provisioned
     IF    ${need_reset} == ${TRUE}
         ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
         Reset To Default Secure Boot Keys    ${advanced_menu}
+        Sleep    3s
```

Comment:
Any comment on this?

Reply:
@macpijan Related to https://github.com/Dasharo/open-source-firmware-validation/issues/154

Reply:
Probably tests failed here when running `Exit From Current Menu` KWD after `Press Enter` from `Reset To Default Secure Boot Keys`. I would say that we did not found better solution to make sure that exiting can be run.